benchmark/src/binfhe-ap.cpp

```diff
@@ -149,8 +166,10 @@ void FHEW_KEYSWITCH(benchmark::State& state, ParamSet param_set) {
     }
 }
 
+BENCHMARK_CAPTURE(FHEW_KEYSWITCH, MEDIUM, MEDIUM)->Unit(benchmark::kMicrosecond)->MinTime(1.0);
+
 BENCHMARK_CAPTURE(FHEW_KEYSWITCH, STD128, STD128)->Unit(benchmark::kMicrosecond)->MinTime(1.0);
 
 BENCHMARK_CAPTURE(FHEW_KEYSWITCH, STD128_AP, STD128_AP)->Unit(benchmark::kMicrosecond)->MinTime(1.0);
 
-BENCHMARK_MAIN();
+BENCHMARK_MAIN();
```

Comment:
Add a new line to the end of the file. 
there is an explanation to this: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Reply:
Addressed. Thanks for the information!